docs/guides/build-using-too-many-resources.rst

```diff
@@ -23,3 +23,20 @@ If there are extra packages that you don't need for building docs,
 you can create a custom requirements file just for documentation.
 This should speed up your documentation builds,
 as well as reduce your memory footprint.
+
+Use pip when possible
+---------------------
+
+In case you are using `conda` to create your environment,
+you could consider using `pip` instead since `conda` could require too much memory to calculate the dependency tree.
```

Comment:
I feel like this sentence needs some sort of supporting sentence. Why does conda use more memory than pip? Also, I'd phrase it as "If you don't need conda to create your environment, ..."

Reply:
I don't know the exact reason of why this is true, but it behaves like that. Also, there are different issues reported at different repositories about this problem of conda consuming too much memory. An example in the official repo: https://github.com/conda/conda/issues/5003

I agree with you that we need to be more explicit here, but I need to think about how to rewrite it.